### PR TITLE
EAHW-2241: Deploy basic accruals restapi

### PR DESCRIPTION
**Pull request by joannafawl**
- Created a basic accruals service with a Hello World endpoint (which will be deleted later)
- Added deployment steps to drone.yml

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
...MHlOekE0TWpneApPVFU0TlRSYU1CVXhFekFSQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBCkE0SUJEd0F3Z2dFS0FvSUJBUURQNm5ZVGpmbmR2T2kxOEltYm5idk9aOXJ3c1hxSkZFTE1IVTkxT2VIaXFIT2kKT1luVFZ3cmpCaDJpZExUM0ozbkVoYnM0WFRIcVlwcVEzdmcwVzluVVRPMnk5ZmFmdlRZVTk0NGZXUTl1SXJOZwo1Y0Q0SXhzK05vWTJmdkkrcStYUUM2SFNiRGJ0S0dOUy9Kbmc5SFdsazF2dzY0TlBwbU5IMTJtUFVvNGpGZnc5CmZoL3RwY1k5akVsZnhJckJBVEJzNHhYcDJHOFFrK1ZCSWE5bnF4bmtVTHJvT0t2Y092T3g4RDU1TXdXcEVKc3QKdENZSXFHM05xdlhMcUdTQzdueHoxYmJNSmZjRDdBRmdvZnM1bXZ0bmlRTHBnZG93WmxNc3g2cnJVM2pBUkw3OApvTktXR3p3c3BKNjhybXdhSGZ4NlRBWldHbUxSU2NPd0xlbWJ3cDRIQWdNQkFBR2pJekFoTUE0R0ExVWREd0VCCi93UUVBd0lCQmpBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElCQVFDb2RrZkMKS2NOYzY5cXR6ZllWUUFlajFVb0hhMjhUUzlqMHV3RVdqWlU0dFVWR2cvZDNCK2YvcnYyN2tqRGM3U2JJSForVQpxMlpMVlVnYU85OVoyUjQyUFhNVTZCRUVUdjUvT0t4amdkNlFNQnFFVkxBdU1XZFpsbnBkMEpDMDFGd0tJb3BGCnVpVUFoWnRjR0tJRXg1U1N1SUlKdFU4a2pnR2xuS0V6aXJrUVVCOUJUOURSQ1VhelQ1ZUp6VFVLSVliWjBNL3gKSUZMbUQvZmZZeFhmRGlSM25YWkdZSWpkUGJHYW42ZU5pZExjQkdEZkxabzZFdkJTR2FUSHl5KzEyc3c2clhTVwpBTVFWd3BUY2ZpSlM1NG9xSXhDU3dyam5XNFZYRnhiSC96SWZXMGs5M29ucERNdXhjT20rN0NuZjViRlJEdlpzCjRCaEV1ank1eEVtajA1TWwKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```
Comment (JWeston-HO) [inline .drone.yml:76]: Should we store this as a secret?

Comment (joannafawl) [reply .drone.yml:76]: I don't think it needs to be as it's the public key for the cluster

**Review by claire-guest — APPROVED** [review]
Approving after walkthrough with Joanna

**Review by daleelR — APPROVED** [review]
Approving java & drone side.

**Review by laurieboyes — APPROVED** [review]
Cool 😎

**Pull request merged**